functions.sh.in

```diff
@@ -96,10 +96,13 @@ _print_repository_path() {
 }
 
 repo_to_repositories() {
-	local func path
+	local func path _repo upstream_file upstreams
```

Comment:
I thought that the idea was to no use underscore prefixed variables within abuild itself, so it is safe to use underscored in  APKBUILDs. How would we otherwise avoid collisions?

This is similar to what arch linux does:
> If you need to create any custom variables for use in your build process, it is recommended to prefix their name with an _ (underscore). This will prevent any possible name clashes with internal makepkg variables. For example, to store the base kernel version in a variable, use something similar to $_basekernver.

https://www.archlinux.org/pacman/PKGBUILD.5.html#_options_and_directives

Reply:
That makes complete sense. I changed functions to not use _ prefix.